src/game_inventory.cpp

```diff
@@ -665,6 +665,22 @@ class comestible_inventory_preset : public inventory_selector_preset
         const player &p;
 };
 
+static std::string get_consume_needs_hint( player &p )
+{
+    const auto &cmgr = get_all_colors();
+    auto hint = std::string();
+    auto desc = p.get_hunger_description();
+    hint.append( string_format( "[%s <color_%s>%s</color>] ", _( "Food :" ),
```

Comment:
Two minor points:

It might be cleaner to use the `colorize` function here rather than writing the tags manually in the format string.  Then you don't need to bother with `get_all_colors`.

Also, it might be worth translating this format string just in case another language wants things formatted differently.

Reply:
@jbytheway oh, nice, I did not see this `colorize` function, I will make this change, this will make the code cleaner.

Also, about:
> translating this format string just in case another language wants things formatted differently

Nothing do do here, it should already be translated since I am using the `_( std::string )` function, just like its done in the panels, and also `get_hunger_description` , `get_thirst_description` and `get_pain_description` returns already translated strings.

But, maybe you are suggesting to use a big parametrized string, all in one for the 3 needs ?

Reply:
Should I push in this already merged PR though or open a new one for this ?